kobo-uncaged/main.go

```diff
@@ -70,18 +71,27 @@ func returncodeFromError(err error, k *device.Kobo) returnCode {
 	return rc
 }
 func mainWithErrCode() returnCode {
-	w, err := syslog.New(syslog.LOG_DEBUG, "KoboUNCaGED")
+	kuLogFile := os.Getenv("KU_LOGFILE")
+	var w io.Writer
+	var err error
+	log.SetPrefix("[Kobo-UNCaGED] ")
+	if kuLogFile != "" {
+		w, err = os.OpenFile(kuLogFile, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
```

Comment:
Are you going to limit the size or rotate the log files?

Reply:
Nevermind, I just noticed the `rm` in the start script.

Reply:
Yeah, I'm letting the startup script manage the log file lifecycle.